misc/about-uris.md

```diff
@@ -0,0 +1,23 @@
+---
+note: ekstra notater om URI-er som ble kuttet.
+---
+
+# Om URI-er i RDF
+
+Ressurser som beskrives i RDF navngis ved hjelp av en unik, global identifikator, en såkalt URI ("Uniform Resource Identifier").
+Strengt tatt bruker man IRI-er ("Internationalized Resource Identifier") i RDF, men begrepene URI og IRI brukes ofte om hverandre som synonymer.
+
+Ofte vil URI-er til en ressurs være automatisk opprettet, slik at du slipper å gjøre dette selv. Dere bør tenke på at URI-er ideelt sett skal være unike og varige, og ideelt sett peke til faktiske nettressurser.
+Navnerommet til en ressurs bør være avgrenset til et område virksomheten har styring på, og ideelt sett ikke bundet opp til navnet på virksomheten eller andre flyktige begrep.
```

Comment:
Er dette dekket av ressursene det er lenket til? Kan være verdifullt å fremheve.

Reply:
Det står litt om det i veilederen til dcat-ap-no. Kan ev. skrive om det i en artikkel om RDF, og/eller sørge for at det dekkes i dcat-ap-no-veilederen. Enig i at det er verdt å ha med, men kanskje ikke så tidlig som på Kom i gang-sidene.